### ISSUE TITLE
Bluetooth simultaneous connections are unstable

### ISSUE THREAD

**Issue by pachnes** [gus33000/MSM8994-8992-NT-ARM64-Drivers#51]
Hi Gus, you should advise the users, to use a cheap blueetooth keyboard with touchpad to use the device after initial configuration. There are a few keyboards with touchpad available at a price between 20,-- and 35,-- Euros. I do not recommend to use separate bluetooth keyboards and separate bluetooth mice, as bluetooth still is not very stable, A single bluetooth device is quiet stable.

**Comment by lwlsw** [comment]
That's right, the current Bluetooth is very unstable, and it is Occasionally disconnect. I use a Bluetooth mouse and a Bluetooth keyboard. They are Occasionally disconnect together,  Also hope to fix it

**Comment by woachk** [comment]
@lwlsw ok, I'll see if there'd be a way to make it more stable, but not too optimistic on that one.

**Comment by lwlsw** [comment]
@woachk 恩，辛苦了，非常感谢您的辛苦付出。

**Comment by SuperJMN** [comment]
@lwlsw Please, write in plain English. Most people won't understand what you say.

**Comment by lwlsw** [comment]
@SuperJMN Ok, I will do in the future.

**Comment by pachnes** [comment]
lslsw did write:
"That's right, the current Bluetooth is very unstable, and it is Occasionally disconnect. I use a Bluetooth mouse and a Bluetooth keyboard. They are Occasionally disconnect together, Also hope to fix it."

That is, why I recommend a bluetooth keyboard device which has an integrated touchpad. If there is only 1 bluetooth connection, bluetooth is quiet stable. Also a bluetooth keyboard with an integrated trackball might work quiet stable, but I did not try it.

**Comment by pachnes** [comment]
woachk, or Gus, I did write this at "Issues". I know, that this is the wrong place to suggest something. Are you able to put a new Tab with the name "Discussion" here?

**Comment by gus33000** [comment]
Issue might be related to Windows classic Bluetooth problems since RS5 release. Won't close just yet as I need to confirm if it's Windows OS related, if it is, I will close this issue and advise people to file a bug report in the Feedback Hub or upvote an existing one like this one: https://aka.ms/AA60c18

![image](https://user-images.githubusercontent.com/3755345/64484592-df5b6b00-d214-11e9-81e3-5e1236c18aa9.png)

**Comment by quiret** [comment]
@gus33000 MSFT might have changed Feedback Hub so that you cannot update "Problem"s anymore. Interesting.

**Comment by falkor2k15** [comment]
Go back to Device manager.
Click on the Bluetooth drop down.
*Right-click on your Bluetooth Adapter choose Properties.
Go to the Power Management Tab.
Uncheck Allow the computer to turn off this device to save power.

*Note: it's called something like "XXX radio service" (the 2nd one in the list)

**Comment by yifengling0** [comment]
> Go back to Device manager.
> Click on the Bluetooth drop down.
> *Right-click on your Bluetooth Adapter choose Properties.
> Go to the Power Management Tab.
> Uncheck Allow the computer to turn off this device to save power.
> 
> *Note: it's called something like "XXX radio service" (the 2nd one in the list)

thanks for your hint。 work but system went unstable( dead with block screen).
and I deleted the driver any more.  Bluetooth working well with no problem. WOW!!

**Comment by gus33000** [comment]
Closing as the issue is windows related
